sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_utils.py

```diff
@@ -217,37 +224,77 @@ def _get_target_for_rpc_dependency(target: Optional[str], attributes: Attributes
     return target
 
 
+# Request
+
+@no_type_check
+def _get_location_ip(attributes: Attributes) -> Optional[str]:
```

Comment:
Is there a specific reason to have the if else logic instead of checking for the migration env var and determining which sem conv is being used. I think I like your approach better, but just wondering.

Reply:
Env var might be at a separate process level. Also, they could be producing some data from each sem conv.